client/my-sites/plan-features/header.jsx

```diff
@@ -87,7 +96,7 @@ export class PlanFeaturesHeader extends Component {
 		const headerClasses = classNames( 'plan-features__header', getPlanClass( planType ) );
 
 		return (
-			<>
+			<span>
```

Comment:
Any reason we need to change it to `<span>` here?

Reply:
If we retain the fragment `<>` instead of using `<span>` , then the tests for the header fails with the following message:

<img width="1435" alt="Screenshot 2019-12-18 at 4 14 29 PM" src="https://user-images.githubusercontent.com/1269602/71079595-928f1500-21b1-11ea-9419-5f6f5cc57c54.png">

Converting to `<span>` seemed like the least amount of change needed to make the tests pass, though it's possible I'm missing something entirely with the way the test is written.